### DISCUSSION TITLE
How to change brush color?

### DISCUSSION THREAD

**Discussion by lomon3** [lllyasviel/Fooocus#2486] [Q&A] [answered]
Hi all. I often work with product images where the product is located on a white background. When I use Inpaint or Outpaint, I don’t see what I painted with a brush.
I will be grateful for any advice)

**Comment by mashb1t** [comment, marked as answer]
This is currently not possible in the user interface, but only the code. You can search for `#FFFFFF` and replace it for inputs you want to change the color with e.g. `#000000` for black.

**Reply by lomon3** [reply]
do you mean in the site code in the browser? or in some file? if in a file, then please tell me in which file?

**Reply by lomon3** [reply]
Fooocus_win64_2-1-831\Fooocus\webui.py
line 200 make brush_color="#F70000"
Thank you)
